apps/dashboard/src/index.css

```
@@ -1,6 +1,148 @@
 @import 'tailwindcss';
+@import 'tw-animate-css';
+@import 'shadcn/tailwind.css';
+@import '@fontsource-variable/inter';
 
-html,
-body {
-  @apply bg-zinc-950 text-zinc-100 min-h-screen;
+@custom-variant dark (&:is(.dark *));
```

Stylelint (17.5.0)

[error] Unexpected unknown at-rule "@custom-variant" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)